src/Alpacka.Lib/Instances/IInstanceHandler.cs

```diff
@@ -14,6 +14,10 @@ public interface IInstanceHandler
         ///           the specified simplified (safe) instance / folder name. </summary>
         string GetInstancePath(string instanceName);
         
+        /// <summary> Gets the full path of an instance for this type with
+        ///           the specified simplified (safe) instance / folder name. </summary>
+        string GetInstancePath(string instanceName, string basedir);
```

Comment:
well all ues of the old one could be changed by passing in the current working directory